File: P1U4.java
package com.mycompany.P1U4;

public class P1U4 {

    public static void main(String[] args) {
        int[] edades = { 17, 19, 18, 17, 18 };
        
        for (int i = 0; i < edades.length; i++) {
            System.out.println("indice: " + i + " es: " + edades[i]);
            
            if (edades[i] >= 18)
                System.out.println("Eres Mayor");
            else
                System.out.println("Eres Menor");
        }
    }
}

File: P1U4.cs
using System;

namespace MyCompany.BaseDeDatos
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] edades = new int[] { 19, 17, 19, 18, 17, };
            int[] edades = new int [] { 19, 17, 18, 18, 19 };

            for (int i = 0; i < edades.Length; i++)
            {
                 Console.WriteLine("indice: " + i + "es: " + edades[i]);
                 if (edades[i] >= 18)
                    Console.WriteLine("Eres Mayor");
                 else
                    Console.WriteLine("Eres Menor");
            }
        }
    }
